README.md

```diff
@@ -59,7 +61,6 @@ The below configuration file will set up the import-map-deployer to do the follo
 {
   "username": "admin",
   "password": "1234",
-  "region": "us-west-2",
```

Comment:
we can use AWS_DEFAULT_REGION env variable instead of providing the region via the json config file.